### PR TITLE
Updated settings menu new location in public-preview-doc-updates.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7330.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/public-preview-doc-updates.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ You can also set the policy using PowerShell using the `CsTeamsUpdateManagementP
 
 To enable the public preview on a desktop or web client, you need to do the following tasks:
 
-1. Select your profile to display the Teams menu.
+1. Select the three dots to the left of your profile to display the Teams menu.
 2. Select **About** → **Public preview**.
```
Comment (JohanFreelancer9) [inline Teams/public-preview-doc-updates.md:50]:
```suggestion
2. Select **About** > **Public preview**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dariomws, not related but a minor additional suggestion, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**